### ISSUE TITLE
Use library math classes

### ISSUE THREAD

**Issue by akofke** [etomica/etomica#33]
Some classes in [etomica.math](https://github.com/etomica/etomica/tree/892bf7d1c6eec322122e6ad72cb28bfc5916b7d0/etomica-core/src/main/java/etomica/math) such as Complex, Quaternion, and possibly Gamma seem to be covered by Apache Commons Math which we already use in core. 

Different from #32 since this is more subjective.

**Comment by ajschult** [comment]
Right.  I think Shu added Complex for Ewald sum; not sure if it's still used there -- we have two implementations!  And Ram added Quaternion for MC moves.  I think his moves still use Quaternions.

Note that for things like the gamma function, we'll want to verify performance.  We were using erfc from SpecialFunctions and then switched to apache commons math because it was more accurate.  However, our own erfc was a lot faster than apache commons and so we're still using that for some applications.
